tests/Nelmio/Alice/Instances/Populator/PopulatorTest.php

```diff
@@ -22,7 +23,7 @@
 class PopulatorTest extends \PHPUnit_Framework_TestCase
 {
     const CONTACT = 'Nelmio\Alice\support\models\Contact';
-    const PLURAL = 'Nelmio\Alice\support\models\PluralProperties';
+    const PROPERTIES = 'Nelmio\Alice\support\models\Properties';
```

Comment:
I would rather explicitly have the child class used than implicitly use the child class for both test cases.